terasoluna-gfw-functionaltest-initdb/pom.xml

```diff
@@ -73,6 +73,7 @@
       <plugin>
         <groupId>org.codehaus.mojo</groupId>
         <artifactId>sql-maven-plugin</artifactId>
+        <version>1.5</version>
```

Comment:
- [ ] Unlike the settings of other plugins, it is a fixed value.
    > Changed the version setting of sql-maven-plugin to be described in initdb so that it is similar to multi-blank. 

    I think you set it the same as a blank project, but why not consider making it a property?

Reply:
You should consider to change or not the blank project and match it. HDYT?
https://github.com/terasolunaorg/terasoluna-gfw-web-multi-blank/blob/50f8eea74797c0d4e1fb0613395f0ded48e22f38/projectName-initdb/pom.xml#L53-L55

And 36fb8b5 looks not good. if use prop, you should manage version at root project.

Reply:
> You should consider to change or not the blank project and match it. HDYT?

I intended to make a similar fix to the blank project.
@btkatoufm 
Is this recognition all right?

> And 36fb8b5 looks not good. if use prop, you should manage version at root project.

I fixed management location of version property.
https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/pull/893/commits/30b978e177565bcaf0e74f45f2c7a8081f7b15bc

Reply:
As a result of the discussion in https://github.com/terasolunaorg/terasoluna-gfw-web-multi-blank/issues/546, the blank project decided to set sql-maven-pluign only in initdb. 36fb8b53c8bb77631a42684f3e025dea1c010596 and 30b978e177565bcaf0e74f45f2c7a8081f7b15bc modifications have been reverted to match gfw-functionaltest as well.
3e4f2115667dba5c713059ed3310e58b104e76d9